tests/reftests/run.ml

```diff
@@ -151,13 +151,17 @@ type filt_sort =
   | GrepV
 
 let escape_regexps s =
+let r =
   let buf = Buffer.create (String.length s * 2) in
   String.iter (function
     | ('|' | '(' | ')' | '*' | '+' | '?'
     |  '[' | '.' | '^' | '$' | '{' | '\\') as c -> Buffer.add_char buf '\\'; Buffer.add_char buf c
     | c -> Buffer.add_char buf c
   ) s;
   Buffer.contents buf
+  in
+  OpamConsole.error "%S -> %S" s r;
+  r
```

Comment:
this section should be reindented. Same for the similar changes in this file

Reply:
oups i left debugging code